samples/template-operator/README.md

```diff
@@ -1,27 +1,59 @@
   * [Custom Reconciliation and Status handling guidelines](#custom-reconciliation-and-status-handling-guidelines)
   * [Local testing](#local-testing)
 * [Bundling and installation](#bundling-and-installation)
-  * [Makefile structure](#makefile-structure)
-  * [Build module operator image](#build-module-operator-image)
+  * [Grafana dashboard for simplified Controller Observability](#grafana-dashboard-for-simplified-controller-observability)
+  * [RBAC](#rbac)
+  * [Build module operator image](#prepare--build-module-operator-image)
   * [Build and push your module to the registry](#build-and-push-your-module-to-the-registry)
-* [RBAC](#rbac)
-* [Grafana dashboard](#grafana-dashboard)
+* [Using your Module in the Lifecycle Manager Ecosystem](#using-your-module-in-the-lifecycle-manager-ecosystem)
+  * [Deploying Kyma Infrastructure Operators with `kyma alpha deploy`](#deploying-the-kyma-infrastructure-operators-with-kyma-alpha-deploy)
+  * [Deploying a `ModuleTemplate` into the control-plane](#deploying-a-moduletemplate-into-the-control-plane)
+  * [Debugging the Operator Ecosystem](#debugging-the-operator-ecosystem)
+  * [Registering your Module within the Control-Plane](#registering-your-module-within-the-control-plane)
 
+## Understanding Module Development in Kyma 
+
+Before going in-depth, make sure you are familiar with:
+
+- [Modularization in Kyma](https://github.com/kyma-project/community/tree/main/concepts/modularization)
+- [Operator Pattern in Kubernetes](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/)
+
+This Guide serves as comprehensive Step-By-Step tutorial on how to properly create a module from scratch by using an operator that is installing a Helm Chart. 
+Note that while other approaches are encouraged, there is no dedicated guide available yet and these will follow with sufficient requests and adoption of Kyma modularization.
+
+Every Kyma Module using an Operator follows 5 basic Principles:
+
+- Declared as available for use in a release channel through the `ModuleTemplate` Custom Resource in the Control-Plane
+- Declared as desired State within the `Kyma` Custom Resource in Runtime or Control-Plane
+- Installed / Managed in the Runtime by Module-Manager through the `Manifest` Custom Resource in the Control-Plane
+- Owns at least 1 CRD that is able to define the Contract that can configure its Behavior
+- Is operating on at most 1 Runtime at every given time
+
+Release channels let customers try new modules and features early, and decide when the updates should be applied. For more info, see the [relevant Documentation in our Modularization Overview](https://github.com/kyma-project/community/tree/main/concepts/modularization#release-channels).
+
+In case you are planning to migrate a pre-existing module within Kyma, please familiarize yourself with the [Transition Plan for existing Modules](https://github.com/kyma-project/community/blob/main/concepts/modularization/transition.md)
```

Comment:
```suggestion
In case you are planning to migrate a pre-existing module within Kyma, please familiarize yourself with the [transition plan for existing modules](https://github.com/kyma-project/community/blob/main/concepts/modularization/transition.md)
```